src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * This subsystem controls the robot intake. It manages
+ * positional control of the intake through a PID loop.
+ * It also controls the intake motors.
+ */
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.configs.CANcoderConfigurator;
+import com.ctre.phoenix6.configs.MagnetSensorConfigs;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.REVLibError;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Intake extends SubsystemBase {
+
+  // local constants
+
+  // IDs
+  private final int intakeMotorID = 10;
+  private final int intakePivotID = 11;
+  private final int intakePivotEncoderID = 9;
+
+  // positions
+  // NOTE: these positions are also used in robotcontainer.
+  public final double intakeSafePosition = 105.73;
+  public final double intakeGroundPosition = 0;
+  public final double intakeLoadPosition = 0;
+
+  // shooter PID
+  private final double intakePValue = 0.2;
+  private final double intakeIValue = 0.0;
+  private final double intakeDValue = 0.0;
+
+  // feed forward PID
+  private final double intakeSValue = 0.0;
+  private final double intakeGValue = 0.0;
```

Comment:
We don't need to clutter our class with constants that will just be 0 for the entire season. It is up to you but I think that a feedback or feedforward controller value should only get a constant if it is non-zero